### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by thiagosoeiro**
This is another scheduling library indeed. Don't get yourselves confused.

TLDR
After some actual coding attempt with Cdule (https://github.com/deepaksinghvi/cdule), it turned out to have a lot of bugs, does not work as expected and it does not feel very mature after this first touch with the library. Even tried to fix some of the bugs in this fork (https://github.com/ClubNFT/cdule/tree/improvements), but eventually I gave up.

Thankfully, https://github.com/rakanalh/scheduler seems to be better. It also had some minor buges and 
areas for improvement which were fixed in this PR and its ready to go now! This is a fork and will be used as import in wallet manager repository.

### CODE REVIEW

**Review by bbiangul — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,25 @@
+module github.com/rakanalh/scheduler
```
Comment (yznima) [inline go.mod]: Can you update this to `github.com/ClubNFT/scheduler`?

Comment (tsoeiroecore) [reply go.mod:1]: Sure, I did that actually but debbuging stopped working and would not stop in the breakpoints. Weird.

**Review by yznima — APPROVED** [review]
